pytket/docs/changelog.rst

```diff
@@ -116,7 +116,7 @@ Fixes:
 
 * Fix `symbol_substitution` not preserving opgroups.
 * Remove hardware inefficient circuit construction in `_tk1_to_rzsx`
-* Support converting conditional `RangePredicate`s to QASM.
+* Support converting conditional `RangePredicate` s to QASM.
```

Comment:
In the other parts of the changelog we use `` instedt of ` , should we do that consistent?

Reply:
```suggestion
* Support converting conditional ``RangePredicate`` s to QASM.

Reply:
As the changelog is now an rst file, can we link to the documentation of the classes / functions in a good way?